simulation_parameters/Constants.cs

```diff
@@ -355,9 +355,9 @@ public static class Constants
     public const bool ALWAYS_SHOW_STORED_COMPOUNDS_IN_REPRODUCTION_PROGRESS = false;
 
     /// <summary>
-    ///   How much total compounds can be absorbed by organelles to grow per second
+    ///   Multiplier on how much total compounds can be absorbed by organelles to grow per second.
     /// </summary>
-    public const float MICROBE_REPRODUCTION_MAX_COMPOUND_USE = 0.5f;
+    public const float MICROBE_REPRODUCTION_MAX_COMPOUND_USE = 2.0f;
```

Comment:
At a hex size of 1, doesn't this now cause the max reproduction speed to be multiple times faster? So doesn't this mean that small cells can now basically ignore the reproduction speed limit? Did other people balance test this?

Reply:
Sorry, I misunderstood the change. So this is now that 2x the free compounds can be used for reproduction, which is pretty much perfect I think.